snap/snapcraft.yaml

```diff
@@ -0,0 +1,17 @@
+name: nudoku
+version: master
```

Comment:
I'm not familiar with snap. But why does the version here say `master` and below the 1.0.0 tarball is used? Is `master` here something different than git master?